ee/tabby-webserver/src/service/mod.rs

```diff
@@ -60,41 +63,31 @@ impl ServerContext {
         }
     }
 
-    async fn authorize_request(&self, request: &Request<Body>) -> bool {
+    async fn authorize_request(&self, request: &Request<Body>) -> (bool, Option<String>) {
```

Comment:
nit: just return `Option<String>`? None means invalid authentication

Reply:
I'm following the logic of how this was implemented before. There were some cases (like for an access token) where it could be authenticated successfully, but we wouldn't be able to determine a user from it. You'll note that there are paths that return `(true, None)` here. I didn't want to change the authentication logic, so I just refactored a little and made it also return the user if possible.